src/app/screens/Publishers/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+import Container from "../../components/Container";
+import PublishersTable from "../../components/PublishersTable";
+
+import { Allowance } from "../../../types";
+import utils from "../../../common/lib/utils";
+import websites from "./websites.json";
+
+function Publishers() {
+  const [data, setData] = useState<Allowance[]>([]);
+  const navigate = useNavigate();
+
+  function navigateToPublisher(id: string) {
+    navigate(`/publishers/${id}`);
+  }
+
+  async function fetchData() {
+    try {
+      const response = await utils.call<{
+        allowances: Allowance[];
+      }>("listAllowances");
+      const allowances = response.allowances.map((allowance) => {
+        if (allowance.enabled && allowance.remainingBudget > 0) {
+          return {
+            ...allowance,
+            badge: {
+              label: "ACTIVE",
+              color: "green-bitcoin",
+              textColor: "white",
+            },
+          };
+        }
+        return allowance;
+      });
+      setData(allowances);
+    } catch (e) {
+      console.error(e);
+    }
+  }
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <Container>
+      <h2 className="mt-12 mb-2 text-2xl font-bold dark:text-white">
+        Your ⚡️ Websites
+      </h2>
+      <p className="mb-6 text-gray-500">
+        Websites where you have used Alby before
```

Comment:
```suggestion
        Some websites where you have used Alby before
```

should we indicate here that it is only a selection?

Reply:
I'm not too fond of "Some websites".